app/components/UI/FiatOnRampAggregator/Views/GetQuotes.js

```diff
@@ -68,22 +75,30 @@ const GetQuotes = () => {
 			<ScreenLayout.Header description="Buy ETH from one of our trusted providers. You’ll be securely taken to their portal without leaving the MetaMask app." />
 			<ScreenLayout.Body>
 				<ScreenLayout.Content>
-					{quotes?.map((quote) => (
-						<View key={quote.providerId} style={styles.row}>
-							<Quotes
-								providerName={quote.providerName}
-								amountOut={quote.amountOut}
-								crypto={quote.crypto}
-								fiat={quote.fiat}
-								networkFee={quote.netwrokFee}
-								processingFee={quote.providerFee}
-								amountIn={quote.amountIn}
-								onPress={() => handleOnPress(quote)}
-								onPressBuy={() => handleOnPressBuy(quote)}
-								highlighted={quote.providerId === providerId}
-							/>
-						</View>
-					))}
+					{quotes.length <= 0 ? (
+						<Text black center>
+							No providers available!
+						</Text>
+					) : (
+						quotes
+							.filter((q) => q.amountIn && q.amountOut)
```

Comment:
I spoke with Kevin and checking that `q.error` is a better option. He already standardized all the error objects. So let's use error field.

Reply:
so is it safe to assume we can filter dataset as following ? 
```suggestion
							quotes.filter((q) => !q.error)
```

Reply:
I'd suggest `quotes.filter(({error}) => !error)`

Reply:
Fixed